pkg/features/kube_features.go

```diff
@@ -69,6 +69,7 @@ const (
 	RotateKubeletServerCertificate featuregate.Feature = "RotateKubeletServerCertificate"
 
 	// owner: @jinxu
+	// beta: v1.8
```

Comment:
was it `alpha` in 1.8?

Reply:
Typos: both version and stage.

It should be 1.7 alpha.

- https://github.com/kubernetes/kubernetes/blob/v1.7.16/pkg/features/kube_features.go#L117
- https://github.com/kubernetes/kubernetes/pull/60159/files#diff-71e3b98f9a6bbf5b8421e26a7ba0c079f397cd8d49abacdad943c66a4f44f03dL101